app/screens/channel/channel.ios.js

```diff
@@ -108,7 +108,10 @@ export default class ChannelIOS extends ChannelBase {
         );
 
         return (
-            <MainSidebar ref={this.mainSidebarRef}>
+            <MainSidebar
+                testID='channel_screen'
```

Comment:
Yeah, really weird. I initially put it in the drawer's [SafeAreaView](https://github.com/mattermost/mattermost-mobile/pull/4535/files#diff-d474c86b609d890cbaeac60f17a15a69R83) (safe approach as Android) but it can't be located.